Lib/unittest/mock.py

```diff
@@ -1242,6 +1244,47 @@ def patched(*args, **keywargs):
         return patched
 
 
+    def decorate_async_callable(self, func):
+        '''This is same as decorate_callable except patched is a coroutine'''
```

Comment:
Please replace docstring with just a comment.
Docstrings are for public API but this class is entirely an implementation detail.

In the comment please add an explicit text like `NB. Keep the method in sync with decorate_callable`.
Add the same comment to `decorate_callable`.